src/App.js

```diff
@@ -8,22 +9,7 @@ import users from './api/users';
 
 const App = () => (
   <div className="App">
-    <h1>Static list of posts</h1>
-
-    <p>
-      <span>posts: </span>
-      {posts.length}
-    </p>
-
-    <p>
-      <span>comments: </span>
-      {comments.length}
-    </p>
-
-    <p>
-      <span>Users: </span>
-      {users.length}
-    </p>
+    <PostList posts={posts} comments={comments} users={users} />
```

Comment:
prepare a new array with all comments and a user in it, don't pass all three arrays